backend/reservation-service/src/main/kotlin/com/ticketqueue/reservation/service/ReservationService.kt

```
@@ -0,0 +1,234 @@
+                            seatNumber = detail.seatNumber,
+                            grade = detail.grade,
+                            price = detail.price
+                        )
+                    }
+                )
+                // DB commit 성공 후에만 Redis 갱신 — 롤백 시 Redis 오염 없음
+                TransactionSynchronizationManager.registerSynchronization(object : TransactionSynchronization {
+                    override fun afterCommit() {
+                        try {
+                            updateHoldSeatsSet(request.scheduleId, request.seatIds)
+                        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)